### PR TITLE
[changed device height to screen height]

### PR DESCRIPTION
**Pull request by Lokendra-rawat**
# Overview

# Test Plan

### CODE REVIEW

**Comment by mmazzarolo** [comment]
@Lokendra-rawat mind adding a description on how/why this change is needed? why just the screen height?

**Comment by netojose** [comment]
I tested here with a smartphone using virtual bottom bar navigation (OnePlus6), and this change makes sense for me.

File: src/modal.tsx [hunk 1]
```diff
@@ -197,7 +197,7 @@ export class ReactNativeModal extends React.Component<ModalProps, State> {
     showContent: true,
     isVisible: false,
     deviceWidth: Dimensions.get('window').width,
```
Comment (voltsonic) [inline src/modal.tsx]: This should be modified to 'screen' as well.

```suggestion
    deviceWidth: Dimensions.get('screen').width,
```

Scenario: user with virtual android buttons set to hidden with app open.

Steps to reproduce:
- User opens app in landscape mode
- User rotates phone to portrait mode
- User then opens a modal and the spacing meant for when the buttons are visible do not show.

**Comment by voltsonic** [comment]
For those who need this fixed for the time being you can extend the class and overwrite those states before the system begins to render the component.

```javascript
class ModalFixed extends Modal {
  constructor(props) {
    super(props);
    const dscrn = Dimensions.get('screen');
    this.state.deviceWidth = dscrn.width;
    this.state.deviceHeight = dscrn.height;
  }
}
```

**Comment by ptgamr** [comment]
Please let get this merge... 

I am having trouble with the modal on Ipad on landscape....

I am not sure if this is a bug of `Dimensions` on the ipad, but this is what happen for me:

Portrait 
```
Dimensions.get('window') = h x w
Dimensions.get('screen') = w x h (correct)
```
Landscape
```
Dimensions.get('window') = w x h
Dimensions.get('screen') = h x w (correct)
```

**Comment by rewieer** [comment]
Solved in 6911ffb :)

**Pull request closed without merging**